src/xmipp/libraries/reconstruction/volume_subtraction.cpp

```diff
@@ -79,22 +79,25 @@ void POCSFourierPhase(const MultidimArray< std::complex<double> > &phase, Multid
 	DIRECT_MULTIDIM_ELEM(FI,n)=std::abs(DIRECT_MULTIDIM_ELEM(FI,n))*DIRECT_MULTIDIM_ELEM(phase,n);
 }
 
-void computeEnergy(MultidimArray<double> &Vdiff, MultidimArray<double> &Vact, double energy)
+void computeEnergy(MultidimArray<double> &Vdiff, MultidimArray<double> &Vact, double energy, bool dontE)
 {
-	Vdiff = Vdiff - Vact;
-	FOR_ALL_DIRECT_ELEMENTS_IN_MULTIDIMARRAY(Vdiff)
-	energy+=DIRECT_MULTIDIM_ELEM(Vdiff,n)*DIRECT_MULTIDIM_ELEM(Vdiff,n);
-	energy = sqrt(energy/MULTIDIM_SIZE(Vdiff));
-	std::cout<< "Energy: " << energy << std::endl;
+	if (dontE != true)
```

Comment:
`if (contidion != true) == if (condition)`
Preferably do check at the calling site